src/packages/emmett-mongodb/src/eventStore/projections/mongoDBInlineProjection.ts

```diff
@@ -0,0 +1,210 @@
+import {
```

Comment:
**COMMENT:** Nothing to see here, Git is not showing that it was just moved from index.ts 🤷‍♂️